themes/bootstrap3/templates/search/searchbox.phtml

```diff
@@ -1,3 +1,21 @@
+<style>
+    .searchForm-inputs {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 0.5rem;
+    }
+
+    .searchForm-inputs .searchForm-query {
+        width: auto;
+        flex-grow: 1;
+    }
+
+    @media screen and (min-width: 768px) {
+        .searchForm-inputs .searchForm-query {
+            flex-grow: 0;
+        }
+    }
+  </style>
```

Comment:
These styles are just an example and would be removed before merge. The effect is to let the main search box grow on mobile widths to take up whatever space it has available, but to still fit the Find button on the same line if there is nothing else in between. It's nice space saver.